app/src/main/webapp/resources/html/licence.html

```diff
@@ -10,14 +10,14 @@ <h2>Expression Atlas Licence</h2>
         </p>
         <cite>
             <p class="icon icon-conceptual" data-icon="l">
-                <a href="https://academic.oup.com/nar/article/50/D1/D129/6438036">Expression Atlas update: gene and protein expression in multiple species</a>
-                (<i>Nucleic Acids Research</i>, 24 November 2021).
+                <a href="https://academic.oup.com/nar/article/52/D1/D107/7442051">Expression Atlas update: insights from sequencing data at both bulk and single cell level</a>
```

Comment:
We are linking the same publication on the homepage, but the date is different there (01 January 2024).
I am also not sure what is the purpose of this publication here.
Should we list all the publications here as well that we have on the home page or just the latest one?